schema/src/main/jade-tables/pipeline_run.table.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "pipeline_run",
+  "columns": [
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "primary_key"
+    },
+    {
+      "name": "pipeline_name",
+      "datatype": "string",
+      "type": "required"
+    }
+  ]
```

Comment:
Would it make sense to have:
* A FK to the `assay` table?
* A reference to the ENCODE pipeline?

We have all the info we need to populate them.

Reply:
Yes, I'd like to see the link to the pipeline, eg encodeproject.org/pipelines/ENCPL002LPE

Reply:
Why to the assay table?  I'm not sure about this.  Other pipelines will not necessarily have any relationship to an Assay.

Reply:
Assay == Experiment in the ENCODE schema, so it'd be a more explicit way to link all the pipeline runs back to their experiment. The experiment ID will be part of the pipeline_run id, but a dedicated column would make things more explicit / give the repo a direct relationship to JOIN on

Reply:
Adding FKs to both of those tables would make sense to me. Especially since  we're already putting in the work to figure out those relationships.